documentation/decisions/adr/00NN-deployment-service.md

```diff
@@ -0,0 +1,99 @@
+# Deployment Strategy
+
+- **Status:** Accepted <!-- REQUIRED -->
+- **Last Modified:** 2023-07-20 <!-- REQUIRED -->
+- **Related Issue:** [187](https://github.com/HHS/grants-equity/issues/187) <!-- RECOMMENDED -->
+- **Deciders:** Lucas Brown, Sammy Steiner, Billy Daly <!-- REQUIRED -->
+- **Tags:** ADR <!-- OPTIONAL -->
+
+## Context and Problem Statement
+
+We need to choose a deployment strategy for the Grants.gov modernization effort that suits our core needs and will host our entire application ecosystem, both the API and frontend layers.
+
+## Decision Drivers <!-- RECOMMENDED -->
+
+- **Reliable:** The chosen deployment strategy should be able to offer uninterrupted application uptime with consistent results.
```

Comment:
AWS SLA commits to [99.99% monthly uptime](https://aws.amazon.com/compute/sla/), so let's say qualify the word `uninterrupted`. Virtually uninterrupted? or just `able to offer at least 99.99% monthly application uptime`.